pkgs/applications/science/electronics/lepton-eda/default.nix

```
@@ -0,0 +1,70 @@
+{ stdenv,
+ lib,
+ pkgconfig,
+ makeWrapper,
+ texinfo,
+ fetchurl,
+ autoreconfHook,
+ guile,
+ flex,
+ gtk2,
+ glib,
+ gtkextra,
+ gettext,
+ gawk,
+ shared-mime-info,
+ groff,
+ libstroke
+}:
+
+let
+  version = "1.9.13-20201211";
+  shortVer = builtins.head (lib.splitString "-" version);
+in
+stdenv.mkDerivation rec {
+
+  pname = "lepton-eda";
+
+  inherit version;
+
+  src = fetchurl {
+    url = "https://github.com/lepton-eda/lepton-eda/releases/download/${version}/lepton-eda-${shortVer}.tar.gz";
```

```suggestion
stdenv.mkDerivation rec {
  pname = "lepton-eda";
  version = "1.9.13-20201211";

  src = fetchurl {
    url = "https://github.com/lepton-eda/lepton-eda/releases/download/${version}/lepton-eda-${builtins.head (lib.splitString "-" version)}.tar.gz";
```

That should work. Didn't test.